examples/k8s-fio/scripts/run-exp.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env sh
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+TS=$(date +%Y%m%d-%H%M%S)
+
```

Comment:
The assumption here is that there is a top dir at `${OUTPUT_BASE}` with the data for various experiments with various sub-directories corresponding to a run of the script or an internal iteration: